src/main/java/com/variaveis/entity/OpcaoParametro.java

```diff
@@ -1,18 +1,100 @@
 package com.variaveis.entity;
 
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="opcao_parametro")
 public class OpcaoParametro {
 
+	@Id
+	@GeneratedValue
+	@Column(name="id_opcao_parametro")
 	private Integer id;
 	
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name="id_parametro")
 	private Parametro paramentro;
 	
+	@Column(name="nome")
 	private String nome;
 	
+	@Column(name="fim_clausula")
 	private String fimClausula;
 	
+	@Column(name="simbolo")
 	private String simbolo;
 	
+	@Column(name="tabela")
 	private String tabela;
 	
```

Comment:
Se este atributo for algo do tipo sim e não, acredito que devemos criar um enum, ou utilizar a anotação do hibernate para true_false